src/tracing/errors.ts

```diff
@@ -56,18 +69,13 @@ export const filterKnownErrors: Required<ClientOptions>['beforeSend'] = (event:
      */
     if (error.message.match(/Loading chunk \d+ failed\. \(error: .+\.chunk\.js\)/)) {
       const asset = error.message.match(/https?:\/\/.+?\.chunk\.js/)?.[0]
-      const entries = [...(performance?.getEntriesByType('resource') ?? [])]
-      const resource = entries?.find(({ name }) => name === asset)
-      const status = resource?.responseStatus
+      if (shouldFilterChunkError(asset)) return null
+    }
 
-      /*
-       * If the status if 499, then we ignore.
-       * If there's no status (meaning the browser doesn't support `responseStatus`) then we also ignore.
-       * These errors are likely also 499 errors, and we can catch any spikes in non-499 chunk errors via other browsers.
-       */
-      if (!status || status === 499) {
-        return null
-      }
+    if (error.message.match(/Loading CSS chunk \d+ failed\. \(.+\.chunk\.css\)/)) {
+      const relativePath = error.message.match(/\/static\/css\/.*\.chunk\.css/)?.[0]
```

Comment:
why do you do relative path here but the full asset path above for JS? also is it easier just to do for any asset? or you want to specifically do this for css/js only?

Reply:
the actual error in sentry is different between the two

Reply:
https://uniswap-labs.sentry.io/issues/4066689493/?project=4504255148851200&query=is%3Aunresolved&referrer=issue-stream&stream_index=6
https://uniswap-labs.sentry.io/issues/4009148084/?project=4504255148851200&query=is%3Aunresolved&referrer=issue-stream&stream_index=13

Reply:
oh that's annoying by the loader